pywt/multidim.py

```diff
@@ -192,7 +192,7 @@ def idwt2(coeffs, wavelet, mode='sym'):
     for rowL, rowH in zip(L, H):
         data.append(idwt(rowL, rowH, wavelet, mode, 1))
 
-    return np.array(data, np.float64)
```

Comment:
opened #106 with dtype testing added.  Ideally we would merge that one first and I can just edit those tests to also check ``np.complex64`` and ``np.complex128`` as well.